client/src/Components/Game/Game.js

```diff
@@ -1,65 +1,58 @@
 import React, { useState, useRef, useEffect } from 'react'
 import ReactDOM from 'react-dom/client'
-import TextArea from '../TextArea/TextArea'
+import TextArea from './TextArea/TextArea'
 import RestartButton from '../RestartButton/RestartButton'
 import GameOptions from '../GameOptions/GameOptions'
 import GameSummary from '../GameSummary/GameSummary'
 import Timer from '../Timer/Timer'
 import { useSelector } from 'react-redux'
 import { createGame } from '../../services/gameService'
-import './Game.css';
+import './Game.css'
```

Comment:
Wow removing the semi colon [🤨](https://emojipedia.org/face-with-raised-eyebrow/)

Reply:
vs code extension did it automatically i think lol